src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommand.cs

```diff
@@ -43,6 +47,12 @@ protected override void Validate()
         {
             throw new CommandException(Resources.FormatMissingOption(_output.LongName));
         }
+
+        // No need to validate --openapi-version, we'll fallback to whatever is configured by
```

Comment:
What do we do if the value is provided but is incorrect?

Reply:
Apparently, we ignore it.  Some sort of user feedback might be helpful.

Reply:
Sure, I can modify the parsing behavior below. We anticipate that this value will be machine provided but an additional check doesn't hurt.